frontend/src/features/projects/dispose/strings.ts

```diff
@@ -0,0 +1,7 @@
+export const clickableTooltip = 'Click to view property details';
```

Comment:
Using this file for storage of long strings with no variable content.

Reply:
Makes me think we may need to start thinking about using a resource file for all string values one day.